src/DynamoCore/Graph/Nodes/DummyNode.cs

```diff
@@ -56,6 +56,13 @@ public DummyNode(int inputCount, int outputCount, string legacyName, XmlElement
             Description = GetDescription();
             ShouldDisplayPreviewCore = false;
 
+            if (OriginalNodeContent != null)
+            {
+                var legacyFullName = OriginalNodeContent.Attributes["function"];
```

Comment:
If the DummyNode is not a DSFunction, this constructor will not get called.

Instead, it will call `LoadNode(nodeElement)` which gets the node type (this is from line 100 of the same file):
```cs
var legacyFullName = OriginalNodeContent.Attributes["type"];
```
Anyway, I'm adding initialization for the property in both constructors.